src/vs/workbench/browser/part.ts

```diff
@@ -22,7 +22,7 @@ export abstract class Part extends Component {
 	private parent: HTMLElement;
 	private titleArea: HTMLElement | null;
 	private contentArea: HTMLElement | null;
-	private partLayout: PartLayout;
+	protected partLayout: PartLayout;
```

Comment:
@sbatten why is this needed? When I find references all of them seem to call `partLayout.layout(dimension)` which is exactly the same as the `layout()` function in `part.ts` does. I would suggest to leave `partLayout` private and call `super.layout(dimension)` instead.